openvdb_points/tools/PointConversion.h

```diff
@@ -979,6 +1074,151 @@ convertPointDataGridGroup(  Group& group,
     group.finalize();
 }
 
+template<typename PositionWrapper, typename InterrupterT>
+inline float autoVoxelSize(const PositionWrapper& positions,
+                           const size_t pointsPerVoxel,
+                           InterrupterT* interrupter,
+                           Vec3R minBounds,
+                           Vec3R maxBounds,
+                           const math::Mat4d targetTransform,
+                           const Index decimalPlaces)
+{
+    using namespace point_conversion_internal;
+
+    if(pointsPerVoxel == 0) OPENVDB_THROW(ValueError, "Points per voxel cannot be zero.");
+
+    float voxelSize(0.1f);
+
+    const size_t numPoints = positions.size();
+
+    // return the default voxel size if we have zero or only 1 point
+
+    if(numPoints <= 1) return voxelSize;
+
+    // if the provided bounds are invalid, try to recompute
+
+    if(minBounds[0] > maxBounds[0] ||
+       minBounds[1] > maxBounds[2] ||
+       minBounds[2] > maxBounds[2])
+    {
+        tbb::blocked_range<size_t> range(0, numPoints);
```

Comment:
This would be easier to check if minBounds and maxBounds were pointer arguments. That sound good to you?